src/Core/Manifest/Module.php

```diff
@@ -127,9 +128,15 @@ public function unserialize($serialized)
      */
     public function activate()
     {
-        $config = "{$this->path}/_config.php";
-        if (file_exists($config)) {
-            require_once $config;
+        global $project;
+        $config = ModuleManifest::config();
+        $configPath = "{$this->path}/_config.php";
+        if (file_exists($configPath)) {
+            require_once $configPath;
+            if ($project !== $config->get('project')) {
```

Comment:
I suggest to move this whole logic out of Module.php and put it in the Kernel code.


```php
/**
     * Include all _config.php files
     */
    protected function bootConfigs()
    {
    	global $project;
    	$projectBefore = $project;
        // After loading all other app manifests, include _config.php files
        $this->getModuleLoader()->getManifest()->activateConfig();
        if ($project && $project !== $projectBefore) {
        	// deprecation, etc
        }
    }
```